### ISSUE TITLE
espcam stream with TCP and receive UDP packets at same time

### ISSUE THREAD

**Issue by superquala** [me-no-dev/AsyncTCP#163]
Hi
I have a question as an amateur developer and it may not relate to this topic but please answer if you know.
is it possible to send video stream from an espcam to a phone via wifi and TCP communication and send some simple commands from phone to esp-cam module with UDP packets at the same time?
a two way communication, one way with TCP and the other way with UDP 
first of all I want to know is it possible or not?
thanks

**Comment by IamBuildDifferent** [comment]
Yes, of course its possible! The only problem would be the slow speed of the esp.
